ERCS/erc-draft_batch_min_sbts.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
```suggestion
---
eip: 7558
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Updated filename and added `EIP` number.